custom_components/vzug/sensor.py

```diff
@@ -124,9 +124,12 @@ class Program(StateBase):
     @property
     def native_value(self) -> StateType | date | datetime | Decimal:
         device = self.coordinator.data.device
-        if device.get("Inactive") == "true":
+        if device.get("Program"):
+            return device.get("Program")
```

Comment:
One small thing I would like to change to be more in line with HA's coding style:
```suggestion
        if program := device.get("Program"):
            return program
```

Reply:
Many thx for the suggestion👍